testing/code/test_source.py

```diff
@@ -129,7 +129,7 @@ def test_source_strip_multiline():
 def test_syntaxerror_rerepresentation():
     ex = pytest.raises(SyntaxError, _pytest._code.compile, "xyz xyz")
     assert ex.value.lineno == 1
-    assert ex.value.offset in (4, 7)  # XXX pypy/jython versus cpython?
+    assert ex.value.offset in (5, 7)  # XXX pypy/jython versus cpython?
```

Comment:
This started to fail with pypy 6.0. Here's the last build on master which worked, using pypy 5.9:

https://ci.appveyor.com/project/pytestbot/pytest/build/1.0.5300/job/r4jil9ajcon4v7mc

Reply:
hmm, can we get a matching list

@arigo @fijal excuse the ping, are you by chance aware of a portable generalization for obtaining the matching offset

Reply:
@arigo thanks for the note, will do